facedancer/proxy.py

```diff
@@ -348,68 +340,45 @@ def _get_libusb_context(cls):
 
         return cls.context
 
+    @classmethod
+    def _handle(cls) -> usb1.USBDeviceHandle:
+        assert cls.device_handle is not None
+        return cls.device_handle
 
```

Comment:
Is there a reason you need this instead of just using `cls.device_handle`? The issue I have with small private helper methods like this is that now you have an extra thing to have to look up when reading the code. Anything less than three lines of code I would really rather have in-line plz tx.